client/cypress/integration/event-manager/login_spec.js

```diff
@@ -0,0 +1,68 @@
+describe('Logging In', function() {
+
+	context('Should be on login page', function() {
+		it('login page view', function() {
+      cy.visit('/login')
+			cy.get('h4').should('contain', 'Login')
+    })
+	})
+
+	context('Submit button should be disabled on empty fields', function(){
+		it('disabled submit button when both fields are empty', function() {
+			cy.contains('Submit').should('be.disabled')
+		})
+	})
+
+	context('Username field should have text type', function(){
```

Comment:
@QaseemLodhi @taleymirza  remove this testcase, its not testing any logic or functionality